reVX/wind_setbacks/wind_setbacks.py

```diff
@@ -0,0 +1,1386 @@
+            Wind regulations table
+        """
+        regs = BaseWindSetbacks._parse_regs(regs_fpath)
+
+        mask = ((regs['Feature Type'] == 'Structures')
+                & ~(regs['Comment'] == 'Occupied Community Buildings'))
+        regs = regs.loc[mask]
+
+        return regs
+
+    @staticmethod
+    def _split_state_name(state_name):
+        """
+        Split state name at capitals to map .geojson files to regulations
+        state names
+
+        Parameters
+        ----------
+        state_name : str
+            State name from geojson files paths with out spaces
+
+        Returns
+        -------
+        str
+            State names with spaces added between Capitals (names) to match
+            wind regulations state names
+        """
+        state_name = ' '.join(a for a
+                              in re.split(r'([A-Z][a-z]*)', state_name)
+                              if a)
+
+        return state_name
+
+    @classmethod
+    def _map_features_dir(cls, structures_dir):
+        """
+        Map structures .geojson files to state based on file name
+
+        Parameters
+        ----------
+        structure_dir : str
+            Path to directory containing microsoft strucutes *.geojsons. Used
+            to identify structures to build setbacks from. Files should be
+            by state
+
+        Returns
+        -------
+        structure_state_map : dict
+            Dictionary mapping state to geojson file path
+        """
+        features_state_map = {}
+        for file in os.listdir(structures_dir):
+            if file.endswith('.geojson'):
+                state = file.split('.')[0]
+                state = cls._split_state_name(state)
+                features_state_map[state] = os.path.join(structures_dir, file)
+
+        return features_state_map
+
+    @staticmethod
+    def _parse_features(structure_fpath, crs):
+        """
+        Load structures from geojson, convert to exclusions coordinate system
+
+        Parameters
+        ----------
+        structure_fpath : str
+            Path to Microsoft .geojson of structures in a given state
+        crs : str
+            Coordinate reference system to convert structures geometries into
+
+        Returns
+        -------
+        structures : geopandas.GeoDataFrame
+            Geometries for structures in geojson, in exclusion coordinate
+            system
+        """
+        structures = gpd.read_file(structure_fpath)
+
+        return structures.to_crs(crs)
+
+    def compute_setbacks(self, structure_fpath, max_workers=None,
+                         layer=None, description=None, replace=False):
+        """
+        Compute structure setbacks for all states either in serial or parallel.
+        Existing setbacks are computed if a wind regulations file was supplied
+        during class initialization, otherwise general setbacks are computed
+
+        Parameters
+        ----------
+        features_dir : str
```

Comment:
great doc strings but incorrect description. probably elsewhere too. 

What kind of files should these be? What extension does the code look for?